libs/blocks/global-navigation/global-navigation.js

```diff
@@ -232,6 +233,14 @@ class Gnav {
     this.el.append(this.elements.curtain, this.elements.topnavWrapper);
   };
 
+  ims = () => {
+    if (window.adobeIMS) {
+      if (window.adobeIMS.initialized) return this.imsReady();
+      return window.addEventListener(CONFIG.imsReady, this.imsReady, { once: true });
+    }
+    return loadIms(this.imsReady.bind(this));
```

Comment:
It might make sense to eventually move this to the utils, but as the global-navigation(+gnav) are the only ones that pass a callback to loadIms so far - we wanted to keep this contained for now, to not further bloat the utils